components/authentication/tenants/sabhardw.yaml

```diff
@@ -0,0 +1,24 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cluster-network-reader-role
+rules:
+  - verbs:
+      - get
+      - list
+    apiGroups:
+      - config.openshift.io
+    resources:
+      - networks
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cluster-network-read-sabhardw
+subjects:
+  - kind: User
+    name: RAMESSESII2
```

Comment:
I think this should be in lowercase?

Reply:
Sorry, again this is just because of misunderstanding of me being a member of PipelineService cluster where I'm "RAMESSESII2"--> changed my Github handle to all lower just recently.